src/zulqer/narcissisticNumbers/index.ts

```diff
@@ -0,0 +1,22 @@
+function isNarcissisticNumber(num: number): boolean {
+  const arrayOfCurrentNumber = String(num).split('');
+  const length = arrayOfCurrentNumber.length;
```

Comment:
Using `length` property from the array every time instead of storing it in a variable is no more a bad way. There is no such performance issue in JS as length is just a property of an array and it won't execute any code to find the length.

Reply:
May be we can still do this when using it too many times just to reduce the variable width.